src/App.tsx

```diff
@@ -65,11 +68,11 @@ function App(): ReactElement {
   // UTILITY METHODS /////////////////////////////////////////////////////////////
 
   /**
-   * Copy the current collection, dataset, feature, track, and frame information
-   * to the page URL.
+   * Get a set of URL parameters that represent the current collection, dataset, feature, track,
+   * and frame information. (Convenience wrapper for `urlUtils.getUrlParams`.)
    */
-  const updateUrl = useCallback((): void => {
-    urlUtils.saveParamsToUrl(
+  const getUrlParams = useCallback((): string => {
```

Comment:
I like this.

Reply:
One nitpick is that the function `getUrlParams` SOUNDS like a function that extracts parameters FROM the url. `stateToUrlParameters` might be closer to what it's really doing?